common/app/layout/Sublink.scala

```diff
@@ -336,6 +336,10 @@ case class ContentCard(
     case Some(_: InlineYouTubeMediaAtom) if !isMediaLink => true
     case _ => false
   }
+
+  val pillar = if (storyContent.isDefined) storyContent.get.metadata.pillar.map(s => s.name.toLowerCase).getOrElse("news")
+
+  val contentType = if (storyContent.isDefined) storyContent.get.metadata.`type`.map(s => s.name.toLowerCase).getOrElse("article")
 }
```

Comment:
Can we have pillar and contentType to be of `Pillar` and `DotcomContentType` rather than strings?

Also it is better to use `map/flatMap` rather than the `if(isDefined) get` pattern:
```
val pillar: Pillar = storyContent.flatMap(_.metadata.pillar).getOrElse(Pillar.News)
val contentType: DotcomContentType = storyContent.flatMap(_.metadata.`type`).getOrElse(DotcomContentType.Article)
```